pkg/cmd/admin/router/router.go

```diff
@@ -287,6 +287,7 @@ func RunCmdRouter(f *clientcmd.Factory, cmd *cobra.Command, out io.Writer, cfg *
 							Spec: kapi.PodSpec{
 								ServiceAccount: cfg.ServiceAccount,
 								NodeSelector:   nodeSelector,
+								HostNetwork: true,
```

Comment:
I think we may need to switch to a different fix here.  I think this is something the endpoints controller should do - going to propose upstream.

> On Jun 18, 2015, at 12:48 AM, Rajat Chopra <notifications@github.com> wrote:
> 
> In pkg/cmd/admin/router/router.go:
> 
> > @@ -287,6 +287,7 @@ func RunCmdRouter(f *clientcmd.Factory, cmd *cobra.Command, out io.Writer, cfg *
> >  							Spec: kapi.PodSpec{
> >  								ServiceAccount: cfg.ServiceAccount,
> >  								NodeSelector:   nodeSelector,
> > +								HostNetwork: true,
> Stat port can be made a unix socket too. Or we can start using multiple interfaces. Though we should eventually pursue this option, I feel cookies is a safer option for a last minute bug fix.
> 
> —
> Reply to this email directly or view it on GitHub.
>